FireRender.Maya.Src/MayaStandardNodesSupport/FileNodeConverter.cpp

```diff
@@ -13,27 +13,77 @@ limitations under the License.
 #include "FileNodeConverter.h"
 #include "FireMaya.h"
 
+#include <sstream>
+
 MayaStandardNodeConverters::FileNodeConverter::FileNodeConverter(const ConverterParams& params) : BaseConverter(params)
 {
 }
 
+void LoadAndAssignUdimImages(const MString& nodeName,  frw::Context context, frw::Image masterImage, const MString& udimPathPattern)
+{
+	MStringArray fileNames;
+
+	std::ostringstream ostream;
+	ostream << "source \"rprCmdRenderUtils.mel\"; geFileNodeUDIMFiles(\"" << nodeName.asChar() << "\")";
+
+	MStatus status = MGlobal::executeCommand(ostream.str().c_str(), fileNames);
+
+	size_t index = std::string(udimPathPattern.asChar()).find("<UDIM>");
+	size_t tagLength = std::string("UDIM").length();
+
+	if (index == std::string::npos)
+	{
+		MGlobal::displayWarning(MString("UDIM pattern is not recognized: ") + udimPathPattern);
+		return;
```

Comment:
maybe move this check to the top of the function before mel command is called?

Reply:
Done